spec.html

```diff
@@ -42851,17 +43243,33 @@ <h1>
           AllocateSharedArrayBuffer (
             _constructor_: a constructor,
             _byteLength_: a non-negative integer,
+            optional _maxByteLength_: a non-negative integer or ~empty~,
```

Comment:
There's only one invocation of `AllocateSharedArrayBuffer`, and it supplies an arg for the `_maxByteLength_` parameter, so there's no need for `_maxByteLength_` to be optional. (But maybe it's optional to echo `AllocateArrayBuffer`?)